lib/client/browser.js

```diff
@@ -219,6 +219,14 @@ require("./discovery.js");
 					});
 
 					QUnit.log(function(details) {
+						const result = {
+							description: details.message,
+							suite: details.module && [qunitHtmlFile, details.module, details.name, details.message] || [],
+							success: details.result,
+							log: testResult.errors || [],
+							time: new Date().getTime() - timer
+						};
+						karma.result(result);
```

Comment:
I'm not sure if this is the right thing to do.
This would report a test for every assertion within a test, right?

Maybe we can add a switch to enable such a behavior?

Reply:
You're right, I've changed accordingly.
Also, is it OK if I also expose an option for users to decide how the data is sent to karma.

Reply:
@matz3 What do you think?